### PR TITLE
WIP: Added support for 'published' comments in the focal mechanisms within…

### PR DESCRIPTION
**Pull request by jsaul**
… EventListView

### CODE REVIEW

File: libs/seiscomp/gui/datamodel/eventlistview.cpp [hunk 1]
```diff
@@ -1027,8 +1045,13 @@ class FocalMechanismTreeItem : public SchemeTreeItem {
 				setText(config.columnMap[COL_REGION], Regions::getRegionName(lat, lon).c_str()); // Region
 			}
 
+			// add publication marker
 			char stat = objectStatusToChar(fm);
-			setText(config.columnMap[COL_TYPE], QString("%1").arg(stat));
+			setText(config.columnMap[COL_TYPE],
+					QString("%1%2")
```
Comment (gempa-jabe) [inline libs/seiscomp/gui/datamodel/eventlistview.cpp:1051]: Either mix tabs and spaces here to align this and all subsequent lines with `config.columnMap`.

Comment (gempa-jabe) [reply libs/seiscomp/gui/datamodel/eventlistview.cpp:1051]: Or add a new line right after `setText(`:

```c++
setText(
<tab>a,
<tab>b,
<tab>c
);
```

File: libs/seiscomp/gui/datamodel/eventlistview.cpp [hunk 2]
```diff
@@ -3176,6 +3197,23 @@ void EventListView::readFromDatabase(const Filter &filter) {
 		}
 		it.close();
 
+		// fetch comments to be able to later mark published focal mechanisms
+		for (size_t i=0; i<ep.focalMechanismCount(); i++) {
+			_reader->loadComments(ep.focalMechanism(i));
+		}
+
+		/*
```
Comment (gempa-jabe) [inline libs/seiscomp/gui/datamodel/eventlistview.cpp:3205]: Why choosing the slow version? Running `focalMechanismCount` additional single queries will slow down the overall event loading especially with remote databases.

**Comment by jsaul** [comment]
Yes, I understand. That's why this is a PR. ;)

**Pull request closed without merging**